tests/lambda_endpoints/test_indexer_lambda.py

```diff
@@ -27,12 +27,12 @@ def write_to_s3(s3_client):
     # write file to s3
     s3_client.put_object(
         Bucket="test-data-bucket",
-        Key=("imap/swapi/l1/2023/01/imap_swapi_l1_sci-1m_20230724_20230724_v02-01.cdf"),
+        Key=("imap/swapi/l1/2023/01/imap_swapi_l1_sci-1m_20230724_v001.cdf"),
```

Comment:
```suggestion
        Key=("imap/swapi/l1/2023/01/imap_swapi_l1_sci-1min_20230724_v001.cdf"),
```

Did we decide to be more explicit with our time descriptors?

Reply:
I didn't hear any push back from SWAPI when I suggested this to them. I updated it.